src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenScriptTests.cs

```diff
@@ -525,16 +525,17 @@ .maxstack  2
             Assert.True(methodData.Method.ReturnsVoid);
             methodData.VerifyIL(
 @"{
-  // Code size       24 (0x18)
+  // Code size       25 (0x19)
   .maxstack  1
-  .locals init (System.Runtime.CompilerServices.TaskAwaiter V_0)
+  .locals init (System.Runtime.CompilerServices.TaskAwaiter<object> V_0)
   IL_0000:  newobj     "".ctor()""
   IL_0005:  callvirt   ""System.Threading.Tasks.Task<object> <Initialize>()""
-  IL_000a:  callvirt   ""System.Runtime.CompilerServices.TaskAwaiter System.Threading.Tasks.Task.GetAwaiter()""
+  IL_000a:  callvirt   ""System.Runtime.CompilerServices.TaskAwaiter<object> System.Threading.Tasks.Task<object>.GetAwaiter()""
```

Comment:
This is an observable change in run-time behavior of the script, it is calling a different method from a different type. We do not control all implementations for Tasks out there in the world. So, calling a different method might have potential negative effects. Even though this is probably an unlikely scenario, I think we should get a green light from @dotnet/roslyn-compat-council.